front_end/scripts/music.js

```diff
@@ -1,40 +1,33 @@
 var tune = document.getElementById("backgroundMusic");
 var output = document.getElementById("demo");
 
-// output.innerHTML = slider.value;// this line should show the default value of slider
-
-
-
-//this code should update the current value of slider based on position 
-try{
+try {
   var slider = document.getElementById("volume");
-  slider.value = sessionStorage.getItem("audioVolume")*100;
-  slider.oninput = function() {
-    let percentageVolume = this.value/100;
+  slider.value = sessionStorage.getItem("audioVolume") * 100;
+  slider.oninput = function () {
+    let percentageVolume = this.value / 100;
     tune.volume = percentageVolume;
     sessionStorage.setItem("audioVolume", percentageVolume);
   }
-}catch (e) {}
+} catch (e) { }
 
 function musicStart() {
   tune.currentTime = sessionStorage.getItem("audioLengthSettings");
   tune.volume = sessionStorage.getItem("audioVolume");
   if (sessionStorage.getItem("musicOn") == "muted") {
     tune.autoplay = 0
-    try{
+    try {
       let muteCheckbox = document.getElementById("music-toggle");
       muteCheckbox.checked = true;
-
-    }catch (e) {}
-
+    } catch (e) { }
   } else tune.autoplay = 1;
 }
+
 window.onbeforeunload = function musicUpdate() {
   sessionStorage.setItem("audioLengthSettings", tune.currentTime);
 }
 
-
-function musicControl(){
+function musicControl() {
   clickSound();
   let muteCheckbox = document.getElementById("music-toggle");
   if (muteCheckbox.checked == true) {
```

Comment:
The value of muteCheckbox.checked is already either true or false. There is no reason to check if it equals true.